Controllers/ResidentialController/controller_mosaik.py

```diff
@@ -40,7 +40,27 @@
 
 
 class controlSim(mosaik_api.Simulator):
-    def __init__(self):
+    def __init__(self) -> None:
+        """
+        Constructor for the controlSim class with the mosaik_api.Simulator class as its Parent class

```

Comment:
Read my comment on redundancy in the doc strings, and correct things across files